src/middlewared/middlewared/plugins/system/product.py

```diff
@@ -95,6 +95,12 @@ def version_major(self):
         # major is "23.10"
         return '.'.join(self.version_short().split('-')[0].split('.', 2)[0:2])
 
+    @accepts()
+    @returns(Str('truenas_release_notes_url'))
+    def release_notes_url(self):
+        """Returns the release notes URL for the SCALE version that is currently installed"""
+        return f'https://truenas.com/docs/scale/{self.version_major()}'
```

Comment:
Not sure if we're planning to change the URL to match the code, but currently it is
https://www.truenas.com/docs/scale/scale22.12/

(so and extra "scale" in there, as IIRC `version_major` will just return "22.12")

Reply:
No the documentation team is setting it up differently starting in Cobia. The link will be just /23.10